Dashboard/email_verification.py

```diff
@@ -66,10 +48,27 @@ def send_verification_email(boto_ses_client: ses_client, email: str) -> dict[boo
     return {'success': True, 'request_id': response['ResponseMetadata'].get('RequestId')}
 
 
+def unverify_email(boto_ses_client: ses_client, email: str) -> None:
+    '''Remove a verified email from ses. Upon failure, return None.
+    This is functionality to be seen and used on the frontend, thus
+    failing is unacceptable.'''
+    if not isinstance(email, str):
+        logging.error(
+            'unverify_email passed `email` argument not of type str.')
+        return None
+    if not is_ses(boto_ses_client):
+        logging.error(
+            'unverify_email client is not a boto3 ses client.')
+        return None
+    logging.info('Removing verified email...')
+    try:
+        boto_ses_client.delete_verified_email_address(
+            EmailAddress=email
+        )
+        logging.info('unverity_email completed successfully, removing email.')
+    except Exception:
```

Comment:
Use a more specific error

Reply:
Same as above, this is client facing, so a catch-all exception is necessary to stop errors reaching customers.